dashboard/test/models/course_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class CourseTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
```

Comment:
Definitely. In practice, I've had `assert true` tests fail.

Rails Interview Question: For what possible reasons might
```
test 'the truth' do
  assert true
end
```
fail?